api/types/system_role.go

```diff
@@ -60,6 +60,9 @@ const (
 	RoleWindowsDesktop SystemRole = "WindowsDesktop"
 	// RoleBot is a role for a bot.
 	RoleBot SystemRole = "Bot"
+	// RoleInstance is a role implicitly held by teleport instances, and is used as a
```

Comment:
What is a Teleport "instance"? A process?

Reply:
That's the million dollar question lol.  Given how graceful restarts work, its technically allowable for multiple teleport processes with the same ID to be running simultaneously.  In this case I'm using the term "instance" to try to refer to the abstract identity represented by an id/data_dir.  Names are hard.

Reply:
That explanation makes sense. I would add it right to the godoc.